### ISSUE TITLE
Actionscript Tracking: Accessibility

### ISSUE THREAD

**Issue by Dinnerbone** [Dinnerbone/ruffle#8]
This is the tracking issue for **Accessibility** in AVM1. We will check off each item of progress as appropriate, and fill in any relevant or missing information as we continue development of Ruffle.
For more information on what each checkbox means, please refer to legend in [the main tracking issue]()

# Accessibility
### Static Method ``isActive`()`
- [ ] Property Exists
- [ ] Has Test Coverage
- [ ] Completely Functional
### Static Method ``sendEvent`(mc, childID, event, isNonHtml)`
- [ ] Property Exists
- [ ] Has Test Coverage
- [ ] Completely Functional
### Static Method ``updateProperties`()`
- [ ] Property Exists
- [ ] Has Test Coverage
- [ ] Completely Functional
